### DISCUSSION TITLE
Exe fails, DLL initialisation error

### DISCUSSION THREAD

**Discussion by gerd35** [pyinstaller/pyinstaller#6142] [Help] [answered]
I have an exe file built in Wine for a PySide project which fails in production (Windows system, restricted user privileges only) because of the following error:

`Traceback (most recent call last):
  File "main.py", line 8, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "PySide6\__init__.py", line 111, in <module>
  File "PySide6\__init__.py", line 60, in _setupQtDirectories
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "shiboken6\__init__.py", line 25, in <module>
ImportError: DLL load failed while importing Shiboken: Eine DLL-Initialisierungsroutine ist fehlgeschlagen.`

Does anyone have an idea what could be the cause for/ solution to this?
The following things have already been tried:
- using PySide2 instead
- using Python 3.8.1 instead of 3.9 for building
- adding pyside6, shiboken6 to hidden imports

**Comment by rokm** [comment, marked as answer]
> I have an exe file built in Wine for a PySide project which fails in production

Does the generated executable run under the original environment in Wine? And does the original, unfrozen version run in that Wine-based environment?

**Reply by gerd35** [reply]
closing this because the person working with me on the project has opened issue 6145
